mettle/api/api.go

```diff
@@ -72,17 +70,23 @@ func serve(opts grpcutil.Opts, name, requestQueue, responseQueue, preResponseQue
 		preResponses: common.MustOpenTopic(preResponseQueue),
 		jobs:         map[string]*job{},
 	}
+
 	go srv.Receive()
+	srv.jobs, err = getExecutions(opts, true)
+	if err != nil {
+		log.Warningf("%s", err)
+	}
+	log.Notice("Updated server with %s inflight executions", len(srv.jobs))
```

Comment:
Also put this in the successful branch, it's a little weird to log it immediately after the warning if you _didn't_ update the executions